docs/05-technical-reference/00-custom-resources/ac-02-eventactivation.md

```diff
@@ -2,7 +2,7 @@
 title: EventActivation
 ---
 
-The `eventactivations.applicationconnector.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to create a Trigger from the Kyma Console and to get an event schema. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
```

Comment:
I changed `Console` to `Dashboard`, but I have doubts as to whether this file is even up to date at all. 
Further down it mentions Console Backend Service and the example is based on ServiceClass ID.
What should we do with it?

Reply:
This file must be removed. `Event Activation` was used by Application Broker.

Reply:
Removed